client/webserver/passbytes.go

```diff
@@ -0,0 +1,188 @@
+// This code is available on the terms of the project LICENSE.md file,
+// also available online at https://blueoakcouncil.org/license/1.0.0.
+
+package webserver
+
+import (
+	"fmt"
+	"strconv"
+	"unicode"
+	"unicode/utf16"
+	"unicode/utf8"
+)
+
+const (
+	escapeSequence        = '\\'
+	unicodePrefix         = 'u'
+	unicodeSequenceLength = 6
+)
+
+// PassBytes is an alias of type []byte that implements json.UnmarshalJSON
+// to enable parsing json-encoded strings as []byte.
+type PassBytes []byte
+
+// UnmarshalJSON satisfies the json.Unmarshaler interface, parses json-encoded
+// strings as []byte. Returns an error if the raw bytes to be unmarshalled is
+// not a valid json-encoded string.
```

Comment:
Here again, it's never technically never a `string`, except in the loosest definition where a "string" refers to any buffer. I think it's okay to talk about "json-encoded bytes". Same in docs for `sanitizeJSONEncodedString`.